suite-native/app/app.config.ts

```diff
@@ -119,6 +119,8 @@ export default ({ config }: ConfigContext): ExpoConfig => ({
             },
         ],
         '@trezor/react-native-usb/plugins/withUSBDevice.js',
+        // Define FLIPPER_VERSION
+        './plugins/withGradleProperties.js',
```

Comment:
shoudn't we apply this plugin only in debug env?

Reply:
Not sure, do you see any problem in having it for all envs? I believe adding gradle properties can be useful in future for something else. For now it should be possible to have it for debug only, but I have to test if the build would work as expected.